plugins/PostProcessingPlugin/scripts/TimeLapse.py

```diff
@@ -66,6 +66,22 @@ def getSettingDataString(self):
                     "type": "float",
                     "default_value": 9000,
                     "enabled": "park_print_head"
+                },
+                "retract":
+                {
+                    "label": "Retraction Distance",
+                    "description": "Filament retraction distance for camera trigger.",
+                    "unit": "mm",
+                    "type": "int",
+                    "default_value": 1
+                },
+                "zhop":
+                {
+                    "label": "Z-Hop when parking",
+                    "description": "Z-hop length before parking",
+                    "unit": "mm",
+                    "type": "float",
+                    "default_value": 0.8
```

Comment:
```suggestion
                    "default_value": 0
```

Same as before; Should be zero or an upgrade script needs to be made.